internal/pkg/util/interactive/interactive_test.go

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Comment:
Same here about the copyright

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```